components/search_results/search_results.jsx

```diff
@@ -209,15 +211,46 @@ export default class SearchResults extends React.PureComponent {
             }
         }
 
+        var formattedTitle = (
+            <FormattedMessage
+                id='search_header.results'
+                defaultMessage='Search Results'
+            />
+        );
+
+        if (this.props.isMentionSearch) {
+            formattedTitle = (
+                <FormattedMessage
+                    id='search_header.title2'
+                    defaultMessage='Recent Mentions'
+                />
+            );
+        } else if (this.props.isFlaggedPosts) {
+            formattedTitle = (
+                <FormattedMessage
+                    id='search_header.title3'
+                    defaultMessage='Flagged Posts'
+                />
+            );
+        } else if (this.props.isPinnedPosts) {
+            formattedTitle = (
+                <FormattedMessage
+                    id='search_header.title4'
+                    defaultMessage='Pinned posts in {channelDisplayName}'
+                    values={{
+                        channelDisplayName: this.props.channelDisplayName,
+                    }}
+                />
+            );
+        }
+
         return (
             <div className='sidebar-right__body'>
                 <SearchResultsHeader
-                    isMentionSearch={this.props.isMentionSearch}
-                    isFlaggedPosts={this.props.isFlaggedPosts}
-                    isPinnedPosts={this.props.isPinnedPosts}
-                    channelDisplayName={this.props.channelDisplayName}
                     isLoading={this.props.isSearchingTerm}
```

Comment:
Could you actually remove the `isLoading` prop? It doesn't seem like it was used anywhere even before this